src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -3454,7 +3454,7 @@ static void Test(int arg1, (byte, byte) arg2)
 }
 ";
             var tree = SyntaxFactory.ParseSyntaxTree(source, options: TestOptions.Regular);
-            Assert.Equal(false, tree.GetRoot().ContainsDiagnostics);
+            tree.GetDiagnostics().Verify();
         }
```

Comment:
>} [](start = 8, length = 1)

Are we testing: `new(int, int)?{}` ?

Reply:
Also test error recovery for: `_ = new (int, int) ? (x) : y;`

---
In reply to: [251128787](https://github.com/dotnet/roslyn/pull/32431#discussion_r251128787) [](ancestors = 251128787)